NDTensors/src/abstractarray/isroc.jl

```diff
@@ -0,0 +1,5 @@
+isroc(A::AbstractArray) = isroc(typeof(A))
+
```

Comment:
Reminder to determine if `iscu` is necessary and delete the functions if we can use `expose` instead

Reply:
I found where iscu is necessary (in a place where adapt was failing due to an issue in the CUDA.jl library) and I made a comment. I'll keep iscu in for now but I removed isroc in favor of using `Expose` system